thorlcr/graph/thgraphslave.cpp

```diff
@@ -1258,6 +1258,23 @@ bool CSlaveGraph::serializeStats(MemoryBuffer &mb)
 
     CRuntimeStatisticCollection stats(graphStatistics);
     stats.setStatistic(StNumExecutions, numExecuted);
+
+    // Track active spills
+    IGraphTempHandler *tempHandler = queryJob().queryTempHandler(false);
+    if (tempHandler)
+    {
+        offset_t activeSpillSize = 0;
+        graph_id gid = queryGraphId();
+        Owned<IFileUsageIterator> iter = tempHandler->getIterator();
+        ForEach(*iter)
+        {
+            CFileUsageEntry &entry = iter->query();
+            if (entry.queryGraphId()==gid)
+                activeSpillSize += entry.getSize();
+        }
+        if (activeSpillSize)
+            stats.setStatistic(StSizeSpillFile, activeSpillSize);
```

Comment:
Not quite sure if this is the spills for this subgraph, or the total active.  I think you are calculating the 1st, but I reporting it as the second.   I may be wrong though!